pkg/registry/core/rest/storage_core.go

```diff
@@ -179,50 +175,90 @@ func (c LegacyRESTStorageProvider) NewLegacyRESTStorage(restOptionsGetter generi
 
 	serviceRest := service.NewStorage(serviceRegistry, endpointRegistry, ServiceClusterIPAllocator, ServiceNodePortAllocator, c.ProxyTransport)
 
-	restStorageMap := map[string]rest.Storage{
```

Comment:
The option may have parsed before, but this group has never actually respected individual enablement.  I don't think we should start now.